### DISCUSSION TITLE
Redis Sorted Sets

### DISCUSSION THREAD

**Discussion by nikalexis** [bmoscon/cryptofeed#832] [Q&A]
Hi, I am trying to use redis backend for orderbooks and snapshots_only=True with Sorted Sets. I see that the Set keeps expanding and thus capturing more and more memory. Is there any way to put a max limit to the entries (max latest orderbook snapshots)?

Thank you!

**Comment by bmoscon** [comment]
you need to manage the data in redis yourself. you should probably have a consumer or something similar that reads and clears data
